build.gradle

```diff
@@ -41,14 +42,17 @@ spotless {
   }
 }
 
+// Use the osdetector-gradle-plugin
```

Comment:
update the comment to say why we need this - "used in combination with the consrypt library"